Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/AzureADConnectHealthFeed.yml

```diff
@@ -0,0 +1,103 @@
+commonfields:
+  id: Azure AD Connect Health Feed
+  version: -1
+fromversion: 5.5.0
+name: Azure AD Connect Health Feed
+display: Azure AD Connect Health Feed
+category: Data Enrichment & Threat Intelligence
+description: Use the Microsoft Azure AD Connect Health Feed integration to get indicators
+  from the feed.
+configuration:
+- display: Fetch indicators
+  name: feed
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Indicator Reputation
+  name: feedReputation
+  defaultvalue: Good
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+- display: Source Reliability
+  name: feedReliability
+  defaultvalue: F - Reliability cannot be judged
```

Comment:
Are we sure about it? don't we want to bump up the reliability?

Reply:
The source is very reliable but, because this is scraping, it would be better to be conservative and let's the users decide